File: plugin/src/org/jetbrains/yesod/julius/highlight/JuliusColors.java
package org.jetbrains.yesod.julius.highlight;

/**
 * @author Leyla H
 */

import com.intellij.openapi.editor.colors.TextAttributesKey;

public interface JuliusColors {
    TextAttributesKey OPERATOR = TextAttributesKey.createTextAttributesKey("JULIUS_OPERATORS");
    TextAttributesKey COMMENTS = TextAttributesKey.createTextAttributesKey("JULIUS_COMMENTS");
    TextAttributesKey ERRORS = TextAttributesKey.createTextAttributesKey("JULIUS_ERRORS");
    TextAttributesKey TEXT = TextAttributesKey.createTextAttributesKey("JULIUS_TEXT");
    TextAttributesKey SIGN = TextAttributesKey.createTextAttributesKey("JULIUS_OPERATORS");
    TextAttributesKey STRING = TextAttributesKey.createTextAttributesKey("JULIUS_STRING");
}

File: plugin/src/org/jetbrains/yesod/julius/highlight/JuliusColors.kt
package org.jetbrains.yesod.julius.highlight

/**
 * @author Leyla H
 */

import com.intellij.openapi.editor.colors.TextAttributesKey

interface JuliusColors {
    companion object {
        val COMMENT: TextAttributesKey = TextAttributesKey.createTextAttributesKey("JULIUS_COMMENT")
        val STRING: TextAttributesKey = TextAttributesKey.createTextAttributesKey("JULIUS_STRING")
        val DOT_IDENTIFIER: TextAttributesKey = TextAttributesKey.createTextAttributesKey("JULIUS_DOTIDENTIFIER")
        val NUMBER: TextAttributesKey = TextAttributesKey.createTextAttributesKey("JULIUS_NUMBER")
        val INTERPOLATION: TextAttributesKey = TextAttributesKey.createTextAttributesKey("JULIUS_INTERPOLATION")
    }
}
